integration/src/main/java/scripts/utils/ClientTestUtils.java

```diff
@@ -50,6 +50,7 @@
 
   public static final String TEST_BUCKET_NAME = "terra_wsm_test_resource";
   public static final String TEST_BQ_DATASET_NAME = "terra_wsm_test_dataset";
+  public static final String TEST_BQ_DATATABLE_NAME = "terra wsm test data table";
```

Comment:
I see you already created this table in the `terra_wsm_test_dataset` dataset - the static resources we use for tests (and user access to them) is documented in the [test users README](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/integration/src/main/resources/testusers/README.md), could you update that too? 
In this case I think that should just be the same permissions as the dataset, as it looks like that's inherited.